scripts/front_end/trias-oauth/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM mysql:5.7
```

Comment:
This docker file is in trias-oauth/, which is assumed to be the 'oauth' service, but it turned out to be mysql service only. In the future, it will confuse people, I suggest creating a new folder named trias-oauth/mysql, and put this Dockerfile and its dependencies into the new folder.